### ISSUE TITLE
Usage of `configure` on an already existing group

### ISSUE THREAD

**Issue by ColonelPanicks** [alces-software/underware#84]
See the below output
```
[root@controller underware]# cat /var/lib/underware/clusters/stucluster/answers/groups/nodes.yaml
---
role: compute
[root@controller underware]# cat /var/lib/underware/clusters/stucluster/cluster-attributes.yaml
---
groups:
- orphan
- login
- nodes
nodes:
  gateway1:
  - login
  node01: &1
  - nodes
  node02: *1
  node03: *1
  node04: *1
  node05: *1
  node06: *1
  node07: *1
  node08: *1
  node09: *1
  node10: *1
[root@controller underware]# bin/underware configure group nodes
error: Too few arguments given. Usage: underware configure group GROUP_NAME NODE_RANGE [options]
```

When configuring an already existing group we require specifying the node range again. This is, in my opinion, problematic as there's a likelihood of incorrectly re-entering the NODE_RANGE.

## Possible solutions

a) Perhaps this could be addressed if there were "meta questions" which are separate to that of `configure.yaml` to ask for the node range. This way it means that the node range won't always need to be re-entered and the "defaults" will be visibly seen on reconfigure.

b) Changing default behaviour for configure if the group already exists such that a NODE_RANGE is not required to go back through the configure questions.

**Comment by WilliamMcCumstie** [comment]
Ahh yes this would be an issue. The same applies for the secondary groups as well. The easiest course forward would  be to make `NODE_RANGE` optional and throw an error if it isn't given with a new group

**Comment by WilliamMcCumstie** [comment]
Fixed in #86
